### ISSUE TITLE
Docker-compose fails: ERROR: manifest for confluentinc/cp-zookeeper:5.4.x-latest not found: manifest unknown

### ISSUE THREAD

**Issue by starkcoffee** [confluentinc/kafka-streams-examples#319]
Hi I just tried to do a docker-up for the music application and I get:
```
5.4.0-post u= ~/projects/kafka-streams-examples: git checkout 5.4.0-post
Already on '5.4.0-post'
Your branch is up to date with 'origin/5.4.0-post'.
5.4.0-post u= ~/projects/kafka-streams-examples: docker-compose up -d
Creating network "kafka-streams-examples_default" with the default driver
Pulling zookeeper (confluentinc/cp-zookeeper:5.4.x-latest)...
ERROR: manifest for confluentinc/cp-zookeeper:5.4.x-latest not found: manifest unknown: manifest unknown
```
I replaced `5.4.x-latest` with `5.4.0` in docker-compose.yml.

**Comment by ableegoldman** [comment]
Why did you replace `5.4.0` with `5.4.x-latest`? Did you try it with `5.4.0` since it's been released?

**Comment by arroadie** [comment]
Should the docker-compose file reflect that?

**Comment by ableegoldman** [comment]
@arroadie guess I should've actually checked the docker-compose, yes it should be `5.4.0`. It looks like our release scripts missed this. I'll submit a PR to fix this on 5.4.0-post right now

@starkcoffee sorry I think I misread your comment. Anyways you need to replace all instances of `5.4.x-latest` with `5.4.0`, based on the error it looks like the zookeeper image at least is still looking for `5.4.x-latest` (which does not exist). I just verified that `docker-compose up -d` works fine on my branch with the image names fixed -- you can go through and replace them all yourself, or just pull the fix once it's merged.

Thanks to both of you for pointing this out!

**Comment by ableegoldman** [comment]
https://github.com/confluentinc/kafka-streams-examples/pull/320
